core/CronArchive/QueueConsumer.php

```diff
@@ -201,6 +201,11 @@ public function getNextArchivesToProcess()
                 break;
             }
 
+            //check if isSegmentAvailable
```

Comment:
That comment is kind of useless as it doesn't provide any additional information than the code below.